tests/blockchain/test_blockchain.py

```diff
@@ -2035,7 +2041,7 @@ async def test_aggsig_garbage(self, empty_blockchain, opcode, with_garbage, expe
         wt: WalletTool = bt.get_pool_wallet_tool()
 
         tx1: SpendBundle = wt.generate_signed_transaction(
-            10, wt.get_new_puzzlehash(), list(blocks[-1].get_included_reward_coins())[0]
+            uint64(10), wt.get_new_puzzlehash(), list(blocks[-1].get_included_reward_coins())[0]
```

Comment:
```suggestion
           10, wt.get_new_puzzlehash(), list(blocks[-1].get_included_reward_coins())[0]
```
Don't seem like this and the change below are related here.

Reply:
it could have been some random mypy issue that pops up from time to time. the mypy version doesn't seem pinned between local runs and CI runs

Reply:
without these changes, pre-commit fails for me:
```
tests/blockchain/test_blockchain.py:2044: error: Argument 1 to "generate_signed_transaction" of "WalletTool" has incompatible type "int"; expected "uint64"  [arg-type]
tests/blockchain/test_blockchain.py:2054: error: Argument 1 to "generate_signed_transaction" of "WalletTool" has incompatible type "int"; expected "uint64"  [arg-type]
```

Reply:
Oh okay yeah seems like it starts checking here now because you added the hinted `consensus_mode: Mode`.